prov/psm2/src/psmx2_fabric.c

```diff
@@ -138,6 +139,7 @@ int psmx2_fabric(struct fi_fabric_attr *attr,
 
 	*fabric = &fabric_priv->util_fabric.fabric_fid;
 	psmx2_active_fabric = fabric_priv;
+	psmx2_trx_ctxt_cnt = 0;
```

Comment:
This is a global, but you're modifying it on the close of a single fabric.  Is this correct?

Reply:
There is only a single instance of fabric at any moment. This global is reset whenever that single instance is created.